doc/stats.rst

```diff
@@ -0,0 +1,106 @@
+`Pairtools stats` as a source of quality control metrics
+===========================================
+
+Overview
+--------
+
+`pairtools stats` produces a human-readable nested dictionary of statistics stored in
+a YAML file or a tab-separated text table (specified through the parameters).
+
+- **Summary statistics** include:
+    - number of pairs (total, unmapped, single-side mapped, etc.),
```

Comment:
Since we have an explicit section called "summary", which only includes fractions and complexity estimations, I think it's confusing to document number of pairs in this section, it should come first. Something like "global statistics" perhaps?